src/rgw/rgw_sal_motr.cc

```diff
@@ -3174,16 +3173,16 @@ int MotrObject::fetch_latest_obj(const DoutPrefixProvider *dpp, bufferlist& bl_o
   string bucket_index_iname = "motr.rgw.bucket.index." + tenant_bkt_name;
 
   keys[0] = this->get_name() + '\a';
-  ldpp_dout(dpp, 20) <<__func__<< ": name=" << keys[0] << dendl;
+  ldpp_dout(dpp, 30) <<__func__<< "DEBUG: name=" << keys[0] << dendl;
```

Comment:
DEBUG should be 20. 
@sachinpunadikar @cdeshmukh @sumedhak27

Reply:
Actually INFO can be @ 20
CRITICAL 0
ERROR  5
WARNING 10
INFO 20
DEBUG 30

Also we cant have Debug logs enabled by default, so default log level needs to limit to 20 / INFO level

Reply:
I am okay with this. 
My thought process was that we only can control log level of Motr SAL, but a S3 request doesn't comes directly to SAL, it goes to RGW frontend, processing module and then SAL. So, I think we should be consistent which how log level works in CEPH codebase in order to restrict logs from all components by setting certain log level.

Reply:
This i agree, but am not sure if 20 log level being used for DEBUG all over, not sure if rest of the team agrees too, we need a discussion on this together.

Reply:
@jjxsg @sjain09 @sachinpunadikar 

> Ceph’s logging levels operate on a scale of 1 to 20, where 1 is terse and 20 is verbose

Going by [Ceph documentation](https://docs.ceph.com/en/latest/rados/troubleshooting/log-and-debug/#ceph-subsystems) also, 20 is DEBUG level. So i agree and can have following log levels mapping to numbers 

ERROR 0
WARNING 5
INFO 10
DEBUG 20

This way we wont need to change existing error logs which are logged using log level 0. But we cant have DEBUG logs enabled by default, which is the case currently. 
If any INFO logs are also logged using 20, need to start change that to 10 and keep only DEBUG at 20.

Also we will introduce macros, atleast in Motr SAL for now, holding these mapping of log levels to number, so tomorrow if/when we want to revisit this, we can just change at one place. Ideally macros needed in common code, but Ceph may/may not accept, though we will try with a Ceph PR & if they agree, we can move these macros from Motr SAL to Common code in future.

If we have some open points still, lets have a discussion on monday.

Reply:
Updated the code with above logic.

Reply:
Agreed with above logic.